inspire_dojson/hep/model.py

```diff
@@ -90,6 +92,18 @@ def copy_special_collections(record, blob):
     return record
 
 
+def move_journal_letters(record, blob):
+    for publication_info in get_value(record, 'references.reference.publication_info', default=[]):
+        journal_title = publication_info.get('journal_title')
+        journal_volume = publication_info.get('journal_volume')
+        if journal_title and journal_volume:
+            if journal_volume[0] in string.uppercase:
```

Comment:
Oh, I didn't know that `string-uppercase` was deprecated in Python 3: https://docs.python.org/3/whatsnew/3.0.html#library-changes
> `string.letters` and its friends (`string.lowercase` and `string.uppercase`) are gone. Use `string.ascii_letters` etc. instead. (The reason for the removal is that `string.letters` and friends had locale-specific behavior, which is a bad idea for such attractively-named global “constants”.)